src/core/config.rs

```diff
@@ -219,4 +219,15 @@ impl Config {
             Environment::None => Err(CliError::MissingEnvironment(env_name.to_owned())),
         }
     }
+
+    /// Determine if an environment is available
+    pub fn has_environment(&self, env_name: &OsStr) -> bool {
+        let env = env_name.to_string_lossy().to_string();
```

Comment:
This is weird, but needed for now as `self.environments` is still keyed with a `String`.

I'm opting not to change `self.environments` to `BTreeMap<OsString, Environment>`
because that breaks `serde_json`. For a fuller explanation, see https://github.com/serde-rs/json/issues/550

Reply:
Yeah, I always try to keep osstring out of code as much as possible. If I have to support it, I usually convert ASAP.